rueidis.go

```diff
@@ -133,7 +133,9 @@ type ClientOption struct {
 	// The default for cluster clients is 0, which means 1 connection (2^0).
 	PipelineMultiplex int
 
-	// ConnWriteTimeout is applied net.Conn.SetWriteDeadline and periodic PING to redis
+	// ConnWriteTimeout is read/write timeout for each connection. If specified,
+	// it is used to control the maximum duration waits for responses to pipeline commands.
+	// Also, ConnWriteTimeout is applied net.Conn.SetWriteDeadline and periodic PING to redis
```

Comment:
```suggestion
	// Also, ConnWriteTimeout is applied net.Conn.SetDeadline and periodic PING to redis
```

Sorry, the original comment was wrong. It should be SetDeadline.

Reply:
[15a95c5](https://github.com/redis/rueidis/pull/469/commits/15a95c59ab9c21c7909e5df35b5054a342cf6046). Yeap. you're right!